.github/DISCUSSION_TEMPLATE/a-setup.yml

```
@@ -0,0 +1,27 @@
+title: "[Setup] "
+labels: ["waiting for support"]
+body:
+- type: dropdown
+  id: problem
+  attributes:
+    options:
```

```suggestion
  attributes:
    label: The problem you are facing
    options:
```

`label` needs to be added here to make it a valid discussion template